README.md

```diff
@@ -58,17 +80,27 @@ await mongoSchedule.disconnect();
+| withCronSchedule | `string`                         | false     |               | Either this setter or `withCronSchedule()` must be called. Specifies the cron schedule according to which the job will run.                                                                                                                                                                                                                                                                                                                                  |
+| withConcurrency  | `number`                         | false     | 1             | How many instances of a job are started at a time.                                                                                                                                                                                                                                                                                                                                                                                                           |
+| withMaxRunning   | `number`                         | false     | 0             | Maximum number of job executions that is allowed at a time. Set to 0 for no max. The schedule will trigger no more job executions if maxRunning is reached. However, there is no guarantee that the schedule always respects the limit; in rare cases with multiple Momo instances maxRunning may be exceeded.                                                                                                                                               |
+| withHandler      | `function`                       | true      |               | The function to execute.                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 
 ### MongoSchedule
 
+You can instantiate a mongo schedule using the `MongoScheduleBuilder` class. It provides the following setter
+methods:
+
+| setter         | parameters    | mandatory | default value | description                                                                     |
+|----------------|---------------|-----------|---------------|---------------------------------------------------------------------------------|
+| withJob        | `MomoJob`     | true      |               | One of the scheduler's jobs. A scheduler must be defined with at least one job. |
```

Comment:
suggestion for description:

Adds a job to the schedule. A schedule must have at least one job.
(NOTE: schedule, not scheduler!)

Btw I am confused that we apparently disallow creating a schedule iwthout a job? Shouldn't that be fine (from technical perspective)? Of course it doesn't make much sense ... but maybe you want to define your jobs later, idk ...

Reply:
Yes, this is exactly a question I wrote down on a post it. :D

Reply:
Yeah then just do it please - I guess it's deleting a `throw Error...` somewhere